metadata-db-app/src/cmr/metadata_db/config.clj

```diff
@@ -6,6 +6,11 @@
    [cmr.oracle.connection :as conn]
```

Comment:
I thought about that and did it in ingest first. but then somehow I thought metadata DB was more generic when dealing with DB entries... I guess not.  Done.